src/main/java/com/iexec/core/task/update/TaskUpdateManager.java

```diff
@@ -377,9 +393,9 @@ void transitionFromRunningState(Task task, ChainTask chainTask) {
             // running2Finalized2Completed must be the first call to prevent other transition execution
             log.debug("Task is running in a TEE without callback, flow contributeAndFinalize is possible [chainTaskId:{}]",
```

Comment:
Shouldn't this log debug message be updated like the comment on line 398 (ex 382) ?

Reply:
fixed in 7e16bda